Resources/views/page_head_style.html.twig

```diff
@@ -5,6 +5,8 @@
     <link rel="stylesheet" type="text/css" href="{{ asset_url }}"/>
 {% endstylesheets %}
 
+<link rel="stylesheet" type="text/css" href="/bundles/ezplatformuiassets/vendors/flag-icon-css/css/flag-icon.min.css" />
```

Comment:
Unfortunately @lserwatka will be at work on Monday so I guess we will just have to wait.